peer/brontide.go

```diff
@@ -4015,18 +4639,32 @@ func (p *Brontide) handleCloseMsg(msg *closeMsg) {
 		return
 	}
 
+	if chanCloserE.IsRight() {
+		// TODO(roasbeef): assert?
```

Comment:
Looks like we need it here so we can send a `ShutdownReceived` event?

Reply:
hmmm looks like the we are missing implementing the msgs handling in the `readHanlder` here for the two new msg types `ClosingComplete` and `ClosingSig`,
https://github.com/lightningnetwork/lnd/blob/8ecef03315f749ae8820b4735978ca2e0552de2e/peer/brontide.go#L2006-L2010